js/songs/factory.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+(function(){
+  angular
+  .module("songs")
+  .factory("RadioFactory", [
+    "$resource",
+    RadioFactoryFunction
+  ]);
+  function RadioFactoryFunction($resource){
+    return $resource("http://localhost:8080/songs/:id", {}, {
```

Comment:
This needs to be the url of the api - which is the Rails API app that is running separately from this Angular app.  O expect that to be on localhost:3000 (or similar).  This angular app, which is just html and js, is running on port 8080.